src/scss/overrides/_variables.scss

```diff
@@ -5,7 +5,7 @@ html {
 $list-item-vertical-margin: 0.25em;
 
 
-$font-family-sans-serif: 'Fira Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+$font-family-sans-serif: 'Zilla Slab', 'Open Sans', X-LocaleSpecific, sans-serif;
```

Comment:
I followed how mozilla.org is doing it. Does this look like a reasonable fallback order? @kristinashu